src/async.js

```diff
@@ -28,6 +29,12 @@ Async.prototype.setScheduler = function(fn) {
     return prev;
 };
 
+// set the batchsize for processing queue
+Async.prototype.setBatchSize = function(batchSize) {
+    ASSERT(batchSize > 0);
```

Comment:
as we discussed, I will update the default batchsize to int_max that will make it essentially no-batching. 
setting batchsize does not require external dispatcher.